queries/goctl/highlights.scm

```diff
@@ -0,0 +1,75 @@
+; Key Symbol
+[
+  "."
+  ","
+  ":"
+  ";"
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+] @punctuation.bracket
+
+; Keywords
+[
+  "syntax"
+  "info"
+  "service"
+] @keyword
+
+"import" @keyword.import
+
+"returns" @keyword.return
+
+[
+  "type"
+  "struct"
+] @keyword.type
+
+[
+  "@doc"
+  "@handler"
+  "@server"
+] @attribute
+
+; Service
+(serviceName) @type
+
+; Httpmethod
+(HTTPMETHOD) @keyword.operator
+
+; Field
+(fieldType) @type.builtin
+
+(fieldName) @variable.member
```

Comment:
I think `(referenceId)` should be here as well

Reply:
I don't think so, (referenceId) can be resolved in special cases like (Class.Field), fieldName is fine for general cases.

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/51357674/964df30b-e5cd-4bc6-9b8e-2d384faf5d88)

Reply:
Oh, I see. What highlight do you think would be appropriate for anonymous fields then? I think those should be highlighted somehow, that was also why I commented this I believe. But maybe there is a more fitting query you know of

Reply:
> Oh, I see. What highlight do you think would be appropriate for anonymous fields then? I think those should be highlighted somehow, that was also why I commented this I believe. But maybe there is a more fitting query you know of

I know what you mean? But now I refer to other highlights on the official website, and basically do not consider the use of different display formats for fieldtypes in different cases, so I prefer to use one color for fieldtypes.

Reply:
I agree. Maybe then just 
```query
(anonymousField) @variable.member
```
?

Reply:
> I agree. Maybe then just
> 
> ```
> (anonymousField) @variable.member
> ```
> 
> ?

yup, as long as it's fieldType we're only going to use one color.